web/src/shared/components/SeqPanel.tsx

```diff
@@ -2,35 +2,38 @@ import * as React from 'react'
 
 import { Accordion } from 'semantic-ui-react'
 import ScienceRoundedIcon from '@mui/icons-material/ScienceRounded'
-import { GraphQlSampleSequencing } from '../../__generated__/graphql'
-import SeqInfo from './SeqInfo'
+import { GraphQlSequencingGroup } from '../../__generated__/graphql'
 import iconStyle from '../iconStyle'
+import { DeepPartial } from '../utilities/deepPartial'
+import SeqGroupInfo from './SeqGroupInfo'
 
 const SeqPanel: React.FunctionComponent<{
```

Comment:
Update name

Reply:
I updated SeqGroupInfo to SequencingGroupInfo - did you want the deepPartial name changed to something too?